app/src/main/java/app/grapheneos/apps/PackageDownloadFgService.kt

```diff
@@ -75,4 +75,9 @@ class PackageDownloadFgService : Service() {
     }
 
     override fun onBind(intent: Intent?): IBinder? = null
+
+    override fun onTimeout(startId: Int, fgsType: Int) {
+        super.onTimeout(startId, fgsType)
+        stopSelf()
```

Comment:
This behavior doesn't fit with the service's purpose

Reply:
`onTimeout` is called after 6 hours of background activity, so would it be best to allow it to throw an exception then? 
It's probably fair to assume that this service shouldn't be running for very long right?